src/components/Common/Filters/Filters.vue

```diff
@@ -233,22 +272,23 @@ export default {
         })
       )
     },
-    onBasicFilterUpdated(filter, sorting) {
+    onBasicFilterUpdated(filter, sorting, isLoadHistory) {
```

Comment:
I don't understand what the variable `isLoadHistory` is for, help me to understand plz :)
Maybe a wording/naming problem?

Reply:
this variable is used not to re-save a search in the history if we launches the search from this one

Reply:
```suggestion
    onBasicFilterUpdated(filter, sorting, loadedFromHistory) {
```